README.md

```diff
@@ -133,16 +133,35 @@ Ubuntu 16.04     | no
 Ubuntu 18.04     | yes
 
 
-## Localization
+## Localization and translation
+
+Zonemaster comes with translation for the following
+languages
+
+* English (en, en_US.UTF-8)
+* French (fr, fr_FR.UTF-8)
+* Swedish (sv, sv_SE.UTF-8)
+* Danish (da, da_DK.UTF-8)
+
+The support of localization or translation is however not complete due to
+complexity in the implementation of `gettext` functions in different OSs.
+The table below documents in what OS the translation function works for
+Engine (i.e. the Zonemaster libraries), the CLI (i.e. the CLI interface
+to the Engine) and Backend (i.e. the RPCAPI interface to the Engine). If
+translation does not work properly in the Backend, it will not work in
+GUI either.
+
+Operating System | Engine | CLI | Backend
+---------------- | -------|-----|--------
+CentOS 7         | ?      |  ?  |   ?
+CentOS 8         | ?      |  ?  |   ?
+Debian 9         | ?      |  ?  |   ?
+Debian 10        | ?      |  ?  |   ?
+FreeBSD 11.3     | yes    | yes | no
```

Comment:
To the best of our knowledge it isn't possible to get translation working in Engine in all circumstances on FreeBSD 11. IMHO we shouldn't say "yes" for Engine here. Maybe "yes*" or something else.

Adding some complexity, I believe the point I make for FreeBSD 12 applies to FreeBSD 11 too.

Reply:
Updated.